From 0aa12de5c1565d56a240d7b0dd814316f4ea81f3 Mon Sep 17 00:00:00 2001
From: Lars Mueller <lmuelle@suse.com>
Date: Fri, 20 Apr 2012 07:59:15 -0400
Subject: [PATCH] mount.cifs: toggle_dac_capability() stores return code

the build process of the cifs-utils for Mandriva 2011 made me notice of
the unused variable rc in toggle_dac_capability() of mount.cifs.c.

A bit up in the code we store the return value and do not make use of it
while calling return.

The attached patch intends to fix this.

The failing build result is still visible at
https://build.opensuse.org/package/live_build_log?arch=x86_64&package=cifs-utils&project=network%3Asamba%3ASTABLE&repository=Mandriva_2011

Acked-by: Suresh Jayaraman <sjayaraman@suse.com>
Signed-off-by: Lars Mueller <lmuelle@suse.com>
---
 mount.cifs.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mount.cifs.c b/mount.cifs.c
index a363b61..06715dd 100644
--- a/mount.cifs.c
+++ b/mount.cifs.c
@@ -577,7 +577,7 @@ toggle_dac_capability(int writable, int enable)
 	}
 free_caps:
 	cap_free(caps);
-	return 0;
+	return rc;
 }
 #else /* HAVE_LIBCAP */
 static int
-- 
1.7.0.4

